Xcode/xcconfigs/Project.xcconfig

```diff
@@ -70,6 +70,10 @@ SYSTEM_HEADER_SEARCH_PATHS                         = $(SRCROOT)/../vendor/SQLite
 
 COMBINE_HIDPI_IMAGES                               = YES    // Stop Xcode from complaining
 
+// Catalyst
+SDKROOT                                            = iphoneos
```

Comment:
Please make sure that this is ok for regular macos.

Reply:
yes. I validated by creating the release package as well as is able to build My Mac/ My Mac(Mac Catalyst) targets successfully from inside Xcode.